agir/notifications/models.py

```diff
@@ -112,3 +112,24 @@ class Meta:
                 name="unique_without_membership",
             ),
         ]
+
+
+class MuteMessage(UUIDIdentified):
+    person = models.ForeignKey(
+        "people.Person",
+        on_delete=models.CASCADE,
+        related_name="muted_person",
+    )
+    message = models.ForeignKey(
+        "msgs.SupportGroupMessage",
+        on_delete=models.CASCADE,
+        related_name="muted_message",
+    )
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["person", "message"],
+                name="unique_person_message",
+            ),
+        ]
```

Comment:
Ce n'était pas plus simple d'ajouter juste une propriété many-to-many (ex. `recipient_blacklist`) au modèle `SupportGroupMessage`, et de faire des `message.recipient_blacklist.add(person)` ou `message.recipient_blacklist.remove(person)` ?

Reply:
Ou alors, il existe déjà le modèle qui lie une personne à un message : `SupportGroupMessageRecipient`,  on pourrait ajouter une propriété `is_muted=True/False`

Reply:
`SupportGroupMessageRecipient` a actuellement l'utilité de marquer la lecture ou non d'un `destinataire` d'un message, cela créerait de la confusion non ?
Pourquoi pas ajouter une blacklist au SupportGroupMessage du coup, mais la solution actuelle me parait plus optimisée avec des relations `one to one` dans une table dédiée ?

Reply:
Ce n'est jamais one to one, vu qu'un personne peut avoir plusieurs messages mutés, et un message peut avoir plusieurs personnes qui l'ont muté. 
Au final, en base de données il aura toujours une table de liaison qui sera créée, mais sans besoin de définir un nouveau modèle et sans besoin des champs en plus (ici le champ UUID pour l'id).
Mais le faire d'ajouter ou enlever une personne de la blacklist d'un message me semble plus clair que de créer ou supprimer une instance de "message muté"